doc/devel/testing.rst

```diff
@@ -40,11 +47,7 @@ particularly useful ones are:
 ``-v`` or ``--verbose``        Be more verbose
 ``-n NUM``                     Run tests in parallel over NUM
                                processes (requires pytest-xdist_)
-``--timeout=SECONDS``          Set timeout for results from each test
-                               process (requires pytest-timeout_)
 ``--capture=no`` or ``-s``     Do not capture stdout
-``--flake8``                   Check coding standards using flake8_
-                               (requires pytest-flake8_)
```

Comment:
I don't think style should be checked via a `pytest` run (we're also not doing this in CI). The test suite already takes a lot of time.

Will add a note on explicit `flake8` runs in the rewrite.